src/components/pages/AdminDashboard/DevTools/DayComponent.js

```diff
@@ -17,6 +17,19 @@ const DayComponent = ({ day, devMode, setDevMode }) => {
     setDevMode(!devMode.isDevModeActive);
   };
 
+  const findNextDayOfWeek = selectedDay => {
+    let dayOfWeek = selectedDay;
```

Comment:
I don't think setting this variable is necessary, since it's just being passed into the function on line 24 later on.  Instead of establishing the variable we could just change 
`(7 + dayOfWeek - date.getDay()) `
to 
`(7 + selectedDay - date.getDay())`
Let me know what you think

Reply:
Good point. I can go and change it.